src/main/java/cms/gov/madie/measure/repositories/MeasureAclRepository.java

```diff
@@ -17,7 +18,11 @@ public interface MeasureAclRepository {
    * @param pageable- instance of Pageable
    * @return Pageable List of measures
    */
-  Page<MeasureListDTO> findMyActiveMeasures(String userId, Pageable pageable, String searchTerm);
+  Page<MeasureListDTO> findActiveMeasures(
```

Comment:
Recommend renaming to better reflect method behavior of performing a measure search.